include/swift/AST/ArgumentList.h

```diff
@@ -71,6 +77,15 @@ class Argument final {
   /// from an ArgumentList.
   void setLabel(Identifier newLabel) & { Label = newLabel; }
 
+  /// The internal label written in the corresponding parameter declaration.
+  Identifier getInternalLabel() const { return InternalLabel; }
```

Comment:
I'm wary of needing this at all. Once the code in SILGen is simplified, you should be able to examine the callee stored in an ApplyExpr and its arguments and form a string in CSApply.

Reply:
Unfortunately the necessary information is lost as it passes through the `DefaultArgumentExpr`. If there's a better way to recover it, please elaborate.